src/pages/About.js

```diff
@@ -0,0 +1,22 @@
+import { Inter } from "next/font/google";
+
+const inter = Inter({ subsets: ["latin"] });
+
+export default function Home() {
+  return (
+    <main>
+      <div class="container mx-auto p-4">
+
+        <div class="space-y-2">
+          <h1 class="text-3xl font-bold tracking-tighter sm:text-5xl xl:text-6xl/none text-[#000080]">
+                About
+          </h1>
+          <p class="max-w-[600px] text-[#333333] md:text-xl">
+                LunaFocus is designed to help you track your cycle.
+          </p>
+        </div>
+
+      </div>
+    </main>
+  );
+}
```

Comment:
ah, thank you for adding this. However, I was envisioning something more like this in the video below..where the links scroll to whatever is on the page..rather than creating a new page ..would you be okay with removing this? I'll follow up with a PR to add what I was thinking?


https://github.com/blackgirlbytes/LunaFocus/assets/22990146/ae73d48a-6759-4083-bb80-3a953cfbbf87

Reply:
Ok, I see what you mean! Ok I think the quickest thing is to close this PR and open a new one with those changes.